<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pizza Order</string>
  </property>
  <property name="accessibleIdentifier">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">/*
Material Dark Style Sheet for QT Applications
Author: Jaime A. Quiroga P.
Inspired on https://github.com/jxfwinter/qt-material-stylesheet
Company: GTRONICK
Last updated: 04/12/2018, 15:00.
Available at: https://github.com/GTRONICK/QSS/blob/master/MaterialDark.qss
*/
QMainWindow {
	background-color:#1e1d23;
}
QDialog {
	background-color:#1e1d23;
}
QColorDialog {
	background-color:#1e1d23;
}
QTextEdit {
	background-color:#1e1d23;
	color: #a9b7c6;
}
QPlainTextEdit {
	selection-background-color:#007b50;
	background-color:#1e1d23;
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: transparent;
	border-width: 1px;
	color: #a9b7c6;
}
QPushButton{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: transparent;
	border-width: 1px;
	border-style: solid;
	color: #a9b7c6;
	padding: 2px;
	background-color: #1e1d23;
}
QPushButton::default{
	border-style: inset;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #04b97f;
	border-width: 1px;
	color: #a9b7c6;
	padding: 2px;
	background-color: #1e1d23;
}
QToolButton {
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #04b97f;
	border-bottom-width: 1px;
	border-style: solid;
	color: #a9b7c6;
	padding: 2px;
	background-color: #1e1d23;
}
QToolButton:hover{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 1px;
	background-color: #1e1d23;
}
QPushButton:hover{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 1px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: #1e1d23;
}
QPushButton:pressed{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #37efba;
	border-bottom-width: 2px;
	border-style: solid;
	color: #37efba;
	padding-bottom: 1px;
	background-color: #1e1d23;
}
QPushButton:disabled{
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #808086;
	border-bottom-width: 2px;
	border-style: solid;
	color: #808086;
	padding-bottom: 1px;
	background-color: #1e1d23;
}
QLineEdit {
	border-width: 1px; border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:#1e1d23;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;
}
QLabel {
	color: #a9b7c6;
}
QLCDNumber {
	color: #37e6b4;
}
QProgressBar {
	text-align: center;
	color: rgb(240, 240, 240);
	border-width: 1px; 
	border-radius: 10px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	background-color:#1e1d23;
}
QProgressBar::chunk {
	background-color: #04b97f;
	border-radius: 5px;
}
QMenuBar {
	background-color: #1e1d23;
}
QMenuBar::item {
	color: #a9b7c6;
  	spacing: 3px;
  	padding: 1px 4px;
  	background: #1e1d23;
}

QMenuBar::item:selected {
  	background:#1e1d23;
	color: #FFFFFF;
}
QMenu::item:selected {
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: #04b97f;
	border-bottom-color: transparent;
	border-left-width: 2px;
	color: #FFFFFF;
	padding-left:15px;
	padding-top:4px;
	padding-bottom:4px;
	padding-right:7px;
	background-color: #1e1d23;
}
QMenu::item {
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: transparent;
	border-bottom-width: 1px;
	border-style: solid;
	color: #a9b7c6;
	padding-left:17px;
	padding-top:4px;
	padding-bottom:4px;
	padding-right:7px;
	background-color: #1e1d23;
}
QMenu{
	background-color:#1e1d23;
}
QTabWidget {
	color:rgb(0,0,0);
	background-color:#1e1d23;
}
QTabWidget::pane {
		border-color: rgb(77,77,77);
		background-color:#1e1d23;
		border-style: solid;
		border-width: 1px;
    	border-radius: 6px;
}
QTabBar::tab {
	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: transparent;
	border-bottom-width: 1px;
	border-style: solid;
	color: #808086;
	padding: 3px;
	margin-left:3px;
	background-color: #1e1d23;
}
QTabBar::tab:selected, QTabBar::tab:last:selected, QTabBar::tab:hover {
  	border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #04b97f;
	border-bottom-width: 2px;
	border-style: solid;
	color: #FFFFFF;
	padding-left: 3px;
	padding-bottom: 2px;
	margin-left:3px;
	background-color: #1e1d23;
}

QCheckBox {
	color: #a9b7c6;
	padding: 2px;
}
QCheckBox:disabled {
	color: #808086;
	padding: 2px;
}

QCheckBox:hover {
	border-radius:4px;
	border-style:solid;
	padding-left: 1px;
	padding-right: 1px;
	padding-bottom: 1px;
	padding-top: 1px;
	border-width:1px;
	border-color: rgb(87, 97, 106);
	background-color:#1e1d23;
}
QCheckBox::indicator:checked {

	height: 10px;
	width: 10px;
	border-style:solid;
	border-width: 1px;
	border-color: #04b97f;
	color: #a9b7c6;
	background-color: #04b97f;
}
QCheckBox::indicator:unchecked {

	height: 10px;
	width: 10px;
	border-style:solid;
	border-width: 1px;
	border-color: #04b97f;
	color: #a9b7c6;
	background-color: transparent;
}
QRadioButton {
	color: #a9b7c6;
	background-color: #1e1d23;
	padding: 1px;
}
QRadioButton::indicator:checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #04b97f;
	color: #a9b7c6;
	background-color: #04b97f;
}
QRadioButton::indicator:!checked {
	height: 10px;
	width: 10px;
	border-style:solid;
	border-radius:5px;
	border-width: 1px;
	border-color: #04b97f;
	color: #a9b7c6;
	background-color: transparent;
}
QStatusBar {
	color:#027f7f;
}
QSpinBox {
	color: #a9b7c6;	
	background-color: #1e1d23;
}
QDoubleSpinBox {
	color: #a9b7c6;	
	background-color: #1e1d23;
}
QTimeEdit {
	color: #a9b7c6;	
	background-color: #1e1d23;
}
QDateTimeEdit {
	color: #a9b7c6;	
	background-color: #1e1d23;
}
QDateEdit {
	color: #a9b7c6;	
	background-color: #1e1d23;
}
QComboBox {
	color: #a9b7c6;	
	background: #1e1d23;
}
QComboBox:editable {
	background: #1e1d23;
	color: #a9b7c6;
	selection-background-color: #1e1d23;
}
QComboBox QAbstractItemView {
	color: #a9b7c6;	
	background: #1e1d23;
	selection-color: #FFFFFF;
	selection-background-color: #1e1d23;
}
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	color: #a9b7c6;	
	background: #1e1d23;
}
QFontComboBox {
	color: #a9b7c6;	
	background-color: #1e1d23;
}
QToolBox {
	color: #a9b7c6;
	background-color: #1e1d23;
}
QToolBox::tab {
	color: #a9b7c6;
	background-color: #1e1d23;
}
QToolBox::tab:selected {
	color: #FFFFFF;
	background-color: #1e1d23;
}
QScrollArea {
	color: #FFFFFF;
	background-color: #1e1d23;
}
QSlider::groove:horizontal {
	height: 5px;
	background: #04b97f;
}
QSlider::groove:vertical {
	width: 5px;
	background: #04b97f;
}
QSlider::handle:horizontal {
	background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
	border: 1px solid #5c5c5c;
	width: 14px;
	margin: -5px 0;
	border-radius: 7px;
}
QSlider::handle:vertical {
	background: qlineargradient(x1:1, y1:1, x2:0, y2:0, stop:0 #b4b4b4, stop:1 #8f8f8f);
	border: 1px solid #5c5c5c;
	height: 14px;
	margin: 0 -5px;
	border-radius: 7px;
}
QSlider::add-page:horizontal {
    background: white;
}
QSlider::add-page:vertical {
    background: white;
}
QSlider::sub-page:horizontal {
    background: #04b97f;
}
QSlider::sub-page:vertical {
    background: #04b97f;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbSize">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>The size of Pizza</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Size</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="rbSmall">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Small size</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Small</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbMedium">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Medium size</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Medium</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbLarge">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Large size</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="accessibleIdentifier">
      <string extracomment="Large"/>
     </property>
     <property name="text">
      <string>Large</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCrustType">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>The type of Crust</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="title">
     <string>Crust Type</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="rbThinCrust">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Thin Crust</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Thin</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbThickCrust">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Thick Crust</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Thick</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbWhereToEat">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>430</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Where to eat</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="title">
     <string>Where To Eat</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QRadioButton" name="rbEatIn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Eat in Resturant</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Eat In</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbTakeOut">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Take the Order Out The Resturant</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Take Out</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbToppings">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Choose Toppings</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Toppings</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QCheckBox" name="chkExtraChees">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Add Extra Chees</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Extra Chees</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkmushrooms">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>84</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Add Mushrooms</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Mushrooms</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkTomatoes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>84</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Add Tomatoes</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Tomatoes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkOnion">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>84</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Add Onion</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Onion</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkOlives">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>90</y>
       <width>84</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Add Olives</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Olives</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkGreenPeppers">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Add Green Peppers</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Green Peppers</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbOrderSummary">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>391</width>
      <height>361</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Order Summary</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="title">
     <string>Order Summary</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>58</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>The size Selected</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Size:</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>The Toppings Selected</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Toppings:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Crust type selected</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Crust Type:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Where you want to eat</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Where To Eat:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>The total price of your order</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string>Total Price:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSize">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Size</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblToppings">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Toppings</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::TextFormat::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblCrustType">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Crust Type</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblWhereToEat">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Where To Eat</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalPrice">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Noto Sans Hebrew Light</family>
       <pointsize>55</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Total Price</string>
     </property>
     <property name="toolTipDuration">
      <number>2000</number>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: red;
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnResetForm">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>610</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Reset your order</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>621</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif Condensed Black</family>
      <pointsize>58</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: green;
}
</string>
    </property>
    <property name="text">
     <string>Make Your Pizza</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOrderPizza">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>610</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Order Pizza Now</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="text">
     <string>Order Pizza</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Close The App</string>
    </property>
    <property name="toolTipDuration">
     <number>2000</number>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
